### DISCUSSION TITLE
entering dates

### DISCUSSION THREAD

**Discussion by mariannevanharten** [fisharebest/webtrees#4613] [Q&A]
It is possible to enter dates as DDMMYYYY (today 22112022).
When dates consist of words FROM and/or TO, the notation doesn't work.
_TO 24111945_
to November 24, 1945
give a strange value...

Best regards,
Marianne

**Comment by fisharebest** [comment]
> TO 24111945
> to November 24, 1945
> give a strange value...

Why is this strange? 24111945 = 24/11/1945 = 24 November 1945
